Reworked ups status messages and fixed tooltip on "Small" footer style

Peuuuur-Noel opened the pull request:
Looks like the Tooltipster js lib was broken due to duplicate "copyright" id in footer code.
Removed #copyright from footer and applied CSS directly to the element.

Peuuuur-Noel committed:
fix: tooltip on "Small" footer style

Peuuuur-Noel committed:
Rework ups status messages

Peuuuur-Noel commented:
I reworked UPS status messages to display exact status reported by UPS/NUT (full list [here](https://networkupstools.org/docs/developer-guide.chunked/ar01s04.html#_status_data)) and not a just the following 4 messages:
```php
'OL'       => 'Online',
'OB'       => 'On battery',
'OL LB'    => 'Online low battery',
'OB LB'    => 'Low battery'
```
I've also add a "severity level" for each messages to change text color.
I defined levels as I thought but it definitely needs to be reviewed ([here](https://github.com/SimonFair/NUT-unRAID/pull/12/commits/85442bf43ab11e01ac286936772528221e63291b#diff-157e0e6413e354961731a97cfbd0349bd595f0f6d816d698becd640c139b131e)).
If you want to force status message, you can uncomment the constant `NUT_STATUS_DEBUG` in [nut_config.php](https://github.com/SimonFair/NUT-unRAID/pull/12/commits/85442bf43ab11e01ac286936772528221e63291b#diff-157e0e6413e354961731a97cfbd0349bd595f0f6d816d698becd640c139b131e) and assign values to test.

I thought of doing this after reading a post from @desertwitch (can't find it anymore) on Unraid Forum.

I let you check that.
Thx.

desertwitch commented:
thanks, looks great and tested working on 6.12.3

ich777 commented:
@Peuuuur-Noel thank you for the changes but I'm not really happy about the tooltip change in the `Small` footer style since I have done this on purpose to make it minimalistic, maybe I should have called the style `Minimal` and to display the text by default in the same color as the copyright.
Sure you'll see the load but that's not that important for me, if I really want to see the load I visit the `Dashboard` or the plugin settings itself.

ich777 commented on source/nut/usr/local/emhttp/plugins/nut/include/nut_footer.php:
```diff
@@ -77,17 +77,30 @@ function format_time($seconds) {
   }
+  $tooltipData = '[' . $nut_name . '] ' . implode(' - ', $online['fulltext']);
+
+  $status[0] = "<span id='" . ($config['FOOTER_STYLE'] == 0 ? "nut_battery" : "") . "' class='tooltip-nut " . $css_class . "' data='" . $tooltipData . "'><i class='fa " . $fa_icon . "' style='vertical-align: baseline;'></i>&thinsp;" . $batteryText . "</span>";
```
@Peuuuur-Noel Maybe `' data='" . $tooltipData` could be included in the check if `FOOTER_STYLE` is `0` to don't show the tooltip when `Small` is enabled if my comment about why it isn't there makes sense to you, the same applies of course to line [126](https://github.com/SimonFair/NUT-unRAID/pull/12/commits/85442bf43ab11e01ac286936772528221e63291b#diff-7b59abd92a7a47a4d9fcabfd2d767ef7343d90c735559552853921584ae40bf2R126)

desertwitch replied:
From the forum:

> This was done on purpose and for two reasons.
When I added this to the plugin I was working on small screens remote and I was constantly hovering over it and it kind of bugged me that this always pops up and the second thing was that it made no sense to me to show a tooltip that shows exactly the same.

I just tested this on my small server screen and the tool-tip really does get in the way, maybe we should disable it in the "small" version so users who don't like it can switch to the "small" version and users who like it can keep the regular "normal" version... so nobody is forced to use/not use the tool-tip.

Peuuuur-Noel replied:
Thanks, I made the change.

desertwitch commented:
Quote from the forum:

> This was done on purpose and for two reasons.
When I added this to the plugin I was working on small screens remote and I was constantly hovering over it and it kind of bugged me that this always pops up and the second thing was that it made no sense to me to show a tooltip that shows exactly the same.

I just tested this on my small server screen and the tool-tip really does get in the way at times. Maybe we should disable it in the "small" version so users who don't like it can switch to the "small" version and users who like it can keep the regular "normal" version... so that nobody is forced to use the tool-tip.

ich777 commented:
@desertwitch it should be also possible to change `Small` to `Minimal` which is in my opinion the better choice, of course I should have done that in the first place but when I introduced the change I had no better name for it.

desertwitch commented:
@ich777: good idea, probably makes more sense to the users that way
@Peuuuur-Noel : can you rename to `Minimal` and disable the tooltip when `Minimal` is used ?

rest of the changes are working fine and looking good
i really like that all status messages are now accounted for both as text and visually by importance
thanks @Peuuuur-Noel for the hard work with the frontend

Peuuuur-Noel committed:
- renamed Small style to Minimal
- disabled tooltip on Minimal style

Peuuuur-Noel commented:
Thanks for your comments.
I changed `Small` to `Minimal` and added a help text to notice that the tooltip is disabled on it.

Do you have any improvements regarding status messages? Is their level good?

desertwitch commented:
> Thanks for your comments. I changed `Small` to `Minimal` and added a help text to notice that the tooltip is disabled on it.
> 
> Do you have any improvements regarding status messages? Is their level good?

Thanks, I think the status messages are good.
I'll test the changes and will report back here.

desertwitch commented:
Thanks for the combined efforts - tested working on 6.12.3
If everyone's happy, I think this is ready to be merged by the master.

Peuuuur-Noel committed:
added code comments

Peuuuur-Noel committed:
typo

Peuuuur-Noel committed:
- if unknown status flag, return it instead of skipping it
- updated TRIM message

Peuuuur-Noel commented:
Ready to merge if you want.
